### PR TITLE
Add chord symbol glyphs to vexflow fonts

### PR DESCRIPTION
**Pull request by AaronDavidNewman**
In support of chord symbols, issue 776.

### CODE REVIEW

File: tools/smufl/config/valid_codes.js [hunk 1]
```diff
@@ -196,4 +196,19 @@ module.exports = {
   vexNoteHeadMutedBreve: 'vf',
   vexNoteHeadRectBlack: 'vd4',
   vexNoteHeadRectWhite: 'vd5',
+
+  csymDiminished:null,
```
Comment (0xfe) [inline tools/smufl/config/valid_codes.js:200]: Few stylistic notes:

- Move this block before the `vex...` blocks.
- Add a comment above the block specifying that these are chord symbols.
- Add a space between the colon and the null

File: src/fonts/bravura_glyphs.js [hunk 2]
```diff
@@ -1,1055 +1,1151 @@
 export const BravuraFont = {
-  'glyphs': {
```
Comment (0xfe) [inline src/fonts/bravura_glyphs.js:2]: You mentioned in the bug that you used a different Bravura OTF file to generate these, can you please add that OTF file to this repo please?

**Comment by AaronDavidNewman** [comment]
>> You mentioned in the bug that you used a different Bravura OTF
When I created the new fork repository, the otf file was correct.  So I must have had a stale version previously.  Style tweaks complete.

**Comment by 0xfe** [comment]
Looks great, thanks. Merging.

**Pull request merged**